tf/common.tf

```diff
@@ -117,3 +117,10 @@ resource "azurerm_key_vault_secret" "sshPubKey" {
   value        = file(pathexpand("~/.ssh/id_rsa.pub"))
   key_vault_id = azurerm_key_vault.main.id
 }
+
+resource "azurerm_key_vault_secret" "rabbitmq-password" {
```

Comment:
hm, the original intention was to keep the azure-swarm repo pure Swarm+Traefik+Portainer and put everything specific to BC or our self-service backend into azure-swarm-bc. But I guess there is no good way for that kind of setup in Terraform or would you have an idea?

Reply:
I already made it optional:
https://github.com/cosmoconsult/azure-swarm/blob/90b16d8b1d7d9f9643a5b5546fba23b282e6b153/tf/common.tf#L122

We could instead create a module out of `azure-swarm` and create a terraform definition for `azure-swarm-bc` which uses the module (https://www.terraform.io/docs/language/modules/index.html).

Reply:
No, that's fair enough, I didn't see the count 👍